### PR TITLE
Add report

### PR DESCRIPTION
**Pull request by dbenders1**
The report is uploaded to the doc directory and the README is changed accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ This ROS package contains the following files:
 
 
 ## Documentation
+### report
+This documents is the resulting report of the project during which this package is created. It explains design, implementation and validation steps of the software architecture and the implemented algorithms. It furthermore provides theoretical conclusions regarding the Active Inference algorithm as well as the Kalman filter.
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
This document is the result of the project during which this package was created. It explains the design, implementation and validation steps of the software architecture and the implemented algorithms. It furthermore provides theoretical conclusions regarding the Active Inference algorithm as well as the Kalman filter.
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ This ROS package contains the following files:
...
+
+This document is useful for people willing to understand the software architecture as well as the theory behind the implemented algorithms.
```
Comment (gavanderhoorn) [inline README.md]:
```suggestion
This document is useful for people looking to understand the software architecture as well as the theory behind the implemented algorithms.
```

**Review by gavanderhoorn — CHANGES_REQUESTED** [review]
Great to see the report added.

I've added some minor suggestions to the text. If you agree with them, you'd only have to click the *Commit suggestion* button.

About the report: I haven't checked, but was that written using Latex?

File: README.md (hunk 2 above)
Comment (dbenders1) [reply README.md]: Yes, the document is written in LateX.

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
:+1:  @dbenders1

**Pull request merged**